.github/workflows/benchmark.yml

```
@@ -38,6 +44,28 @@ jobs:
           key: ${{ runner.os }}-${{ github.sha }}
           restore-keys: ${{ runner.os }}
 
+      - name: Cache test data directory
+        id: cache-test-data
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ env.IRIS_TEST_DATA_PATH }}
+          key:
+            test-data-${{ env.IRIS_TEST_DATA_VERSION }}-${{ env.TEST_DATA_CACHE_BUILD }}
+
+      - name: Fetch the test data
+        if: steps.cache-test-data.outputs.cache-hit != 'true'
+        run: |
+          wget --quiet https://github.com/SciTools/iris-test-data/archive/v${IRIS_TEST_DATA_VERSION}.zip -O iris-test-data.zip
+          unzip -q iris-test-data.zip
+          mkdir --parents ${GITHUB_WORKSPACE}/${IRIS_TEST_DATA_PATH}
+          mv iris-test-data-${IRIS_TEST_DATA_VERSION} iris-test-data
+          mv iris-test-data ${GITHUB_WORKSPACE}/benchmarks
```

```suggestion
          mv iris-test-data-${IRIS_TEST_DATA_VERSION} ${GITHUB_WORKSPACE}/${IRIS_TEST_DATA_PATH}
```

There's presumably no risk that it might already exist in this scenario. If so you could also add `rm -rf ${GITHUB_WORKSPACE}/${IRIS_TEST_DATA_PATH}` first.

---

I need in some way to make sure the parent directory exists before moving the directory. I've done this by introducing a another env variable, which seems cleanest of the things I've thought of but still seems a bit messy